start-gateway-config.sh

```diff
@@ -21,4 +21,4 @@ echo 153 > /sys/kernel/debug/bluetooth/hci0/adv_max_interval
 
 printf "pairable off\nquit" | /usr/bin/bluetoothctl
 
-python3 config_program.py
+python config_program.py
```

Comment:
I think this will break on Balena as `python` doesn't exist (only `python3`).

Reply:
I'm assuming that PR #52 will also be merged and that python will be correct because of virtualenv. Clarified in e4a2dc7.

Reply:
Good point. Yes virtualenv should in theory solve for that.